Dockerfile.backend-container

```diff
@@ -5,7 +5,7 @@ WORKDIR /opt/spire
 ENTRYPOINT ["/opt/spire/run_backend.sh"]
 
 # Add init
-COPY run_backend.sh run_backend.sh
+COPY scripts/run_backend.sh run_backend.sh
 COPY bin/tornjak-backend tornjak-backend
```

Comment:
We should also remove traces of "tornjak-backend" dir right, since taht's main and everything else is components under it?

Reply:
so for within `bin` and in the container, i think having it labeled `tornjak-backend` will make it more clear for debugging purposes. 
this does not affect the root directory structure